terra-jupyter-aou/Dockerfile

```diff
@@ -85,9 +85,7 @@ ENV USER jupyter-user
 USER $USER
 
 RUN pip3 install --upgrade \
-  pandas-profiling==2.10.1 \
```

Comment:
You would also need to make updates to `CHANGELOG.md` and bump the version in `config.json`, see and example: https://github.com/DataBiosphere/terra-docker/pull/200/files

Reply:
done!